app/src/main/java/com/oracle/javafx/scenebuilder/app/util/AppSettings.java

```diff
@@ -136,6 +140,13 @@ public static void getLatestVersion(Consumer<String> consumer) {
                     ex.printStackTrace();
                 }
                 latestVersion = onlineVersionNumber;
+
+                try {
+                    isUpdateAvailable = isCurrentVersionLowerThan(latestVersion);
+                } catch (NumberFormatException e) {
+                    e.printStackTrace();

```

Comment:
Move this to Logger.logSevere ? We shouldn't print stacktraces to console as they get lost unless you run from command line.

Reply:
I was just following what the rest of the code in the block was doing. `e.printStackTrace()` seems to be used quite a lot.

Reply:
I understand, but we have logs for a reason, and that's how new PRs should be tackling exceptions.

Reply:
I did a search of how Logging is being done elsewhere. Seems we have two patterns:-

```
Logger.getLogger(AppSettings.class.getName()).log(Level.SEVERE, "An exception was thrown:", ex);
```

or

```
Logger.getLogger(PreferencesRecordDocument.class.getName()).log(Level.SEVERE, null, ex);
```

With the second one seeming to be a bit more common. Any recommentation on which to use? 

I'm assuming from the output that it will be clear its an exception, making the "An exception was thrown:" text redundant?

Reply:
The `String` message should normally include some useful information. So in the above case it might say: "Cannot parse version string" or something similar.